ts/store/reducers/profile.ts

```diff
@@ -76,6 +76,18 @@ export const profileNameSelector = createSelector(
     )
 );
 
+/**
+ * Return the fiscal code of the profile if some, else undefined
+ */
+export const profileFCSelector = createSelector(
+  profileSelector,
+  (profile: ProfileState): string | undefined =>
+    pot.getOrElse(
+      pot.map(profile, p => p.fiscal_code),
+      undefined
+    )
+);
+
```

Comment:
Is this a function intended for future use? I see it is not used in this pr.

Reply:
At the beginning I thought to use it in `CgnOwnershipInformation.tsx` but I then realized that I needed other more infos about the currentProfile, We can remove this anyway